packages/phone-number-privacy/migrations/20200330212224_create-accounts-table.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex'
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable('accounts', (t) => {
+    t.increments('id')
+      .unsigned()
+      .primary()
+    t.dateTime('createdAt').notNullable()
+    t.string('address').notNullable()
+    t.integer('num_lookups').unsigned()
+    t.dateTime('did_matchmaking')
+  })
+}
+
+export async function down(knex: Knex): Promise<any> {
```

Comment:
Is it possible to limit the privileges of the application identity such that it doesn't have the ability to create/drop tables?

Reply:
Probably possible, I haven't looked into fine grained user controls but we could set up multiple DB users.
It's worth noting that these migrations are not run by the app but one time by us before deploying to app to prime the db

Reply:
Makes sense; thanks for clarifying. Let's see if we can create separate identities for the webapp and the deployment pipeline.